app/views/surveys/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>SurveyCraft</h1>
+
+<% @surveys.each do |survey| %>
+  <% if survey.id %>
```

Comment:
What does this check do?

Reply:
I think I just had a similar issue with my project. It looks like it has to do with re-rendering a template after an invalid submission. This happens when the invalid model is being created as an association. I went with something like this `<% next unless survey.persisted? %>` to show in the code that I'm not displaying records that isn't in the DB yet.